packages/cozy-sharing/src/state.js

```diff
@@ -89,7 +89,9 @@ const updateByIdItem = (state, id, updater) => {
       }
 }
 
+/* eslint-disable default-param-last */
 const indexSharing = (state = {}, sharing) => {
```

Comment:
```suggestion
const indexSharing = (state = {}, sharing) => { // eslint-disable-line default-param-last
```

Reply:
I think we should do that in the whole file 🙏 (it's less verbose than he 2 lines)

Reply:
@trollepierre I guess that makes sense even if I don't like it much. But the file shouldn't really change so disabling the whole file should be ok

Reply:
I am not proposing to disable the whole file. I prefer the current solution rather than disable the whole file.

But I prefer even more using `eslint-disable-line` rather than `eslint-disable` + `eslint-enable`